README.md

```diff
@@ -365,18 +373,27 @@ Another example will show how you can run and do the functional test of the serv
 
 ## Unit testing
 
-Write a unit test for the `calculate_ip_address` function. Start by creating a new folder `tests` inside a python folder of the loopback package. Then create a file inside called `loopback_test.py`. You will use Python built-in Unit test library to implement a unit test that will check if function works correctly and returns correct host address for a given prefix. Add the following code to the created file:
+Write a unit test for the `calculate_ip_address` function. Start by creating a new folder `tests` inside a python folder of the loopback package. Then create a file inside called `test_loopback.py`. 
+
+```
+mkdir ~/src/loopback/python/tests
+touch ~/src/loopback/python/tests/test_loopback.py
+touch ~/src/loopback/python/tests/__init__.py
+```
+
+You will use Python built-in Unit test library to implement a unit test that will check if function works correctly and returns correct host address for a given prefix. Add the following code to the created file:
 
 ```python
-from loopback.ServiceCallbacks import calculate_ip_address
+import unittest
+from loopback.loopback import calculate_ip_address
 
 class TestLoopbackService(unittest.TestCase):
     def test_calculate_ip_address(self):
         self.assertEqual('10.0.0.1', calculate_ip_address('10.0.0.0/24'))
-        self.assertEqual('10.0.0.1', calculate_ip_address('10.0.0.0/24'))
+        self.assertEqual('192.168.10.1', calculate_ip_address('192.168.10.0/24'))
 ```
 
-Execute the unit test by running the following command from the `src/loopback/src` directory:
+Execute the unit test by running the following command from the `~/src/loopback/src` directory:
```

Comment:
is there any reason why the unit test should be run from the src directory inside the loopback package? if there is, can you add that note, it will be useful to know the why.

Reply:
This will be later executed from Makefile that is located here, that is why they are executing it from there.